application/src/test/java/uk/gov/hmcts/dm/service/BlobStorageReadServiceTest.java

```diff
@@ -57,4 +65,18 @@ public void throwsCantReadDocumentContentVersionBinaryException() throws URISynt
 
         blobStorageReadService.loadBlob(documentContentVersion, outputStream);
     }
+
+    @AfterClass
+    public static void killRuntimeAgainstNonDaemonThreadsNotResponding() {
```

Comment:
what are we going to do here?

Reply:
We are just trying to forcefully kill the JVM on which the tests are being executed. This is because after Spring Boot upgrade, an Azure client code is keeping 2-3 non-daemon threads open, which are not exiting and even worse, not responding to the any kill signals, causing Gradle to hang and builds to fail. Thus, we had to force the test suit JVM to halt after a safe period by which the tests are fully executed. This has fixed the issue, after very long efforts to prevent the blocking Thread, with no other solution turning out to be feasible. (The relevant, Microsoft jar's are not open source, by the way.)

Reply:
Comment added to the code. Thank you.